src/main/java/frc/robot/Constants.java

```diff
@@ -11,70 +16,77 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
-    public static final int primaryController = 0;
```

Comment:
IMO it's not a bad idea to go ahead and keep this constant instead of hardcoding the value in `RobotContainer.java()`, but it's not a game changer.